active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,220 @@
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+Number box is a numerics only text control with support for input formatting, validation, basic calculation, and increment stepping.
+
+**Important APIs:** [NumberBox class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.numberbox)
+
+## Is this the right control? 
+
+Use a **NumberBox** control to recieve and display formatted numeric values such as currency, IP addresses, and telephone numbers. A **NumberBox** can also be enabled with basic calculator support to compute multiplication, division, addition, and subtraction across order of operations on entered equations. 
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox"/>
+```
+
+### Add placeholder text and a header
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="simple NumberBox" Header="Enter equation:" PlaceholderText="A + B * C" />
+```
+
+### Enable calculation support
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for simple calculations" 
+    Header="Enter equation:" 
+    PlaceholderText="A + B * C" 
+    AcceptsCalculation="True" />
+```
+
+### Add formatting
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for IP addresss" 
+    Header="Enter equation:" 
+    PlaceholderText="000.000.000.000"
+    FormatType="IPAddress" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for international telephone" 
+    Header="Enter equation:" 
+    PlaceholderText="0 (123) 456-7890"
+    FormatType="InternationalTelephone" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for custom input" 
+    Header="Enter equation:" 
+    PlaceholderText="00 ft 00 in"
+    CustomFormatString="## ft ## in"
+    StripLeadingZeros="False" />
+```
+
+### Add increment and decrement stepping
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for items in order" 
+    Header="Order:" 
+    Text="2"
+    StepSize="2"
+    UpDownArrowsEnabled="True"
```

Comment:
I think it would be better to be more abstract about input devices and not call out arrows in particular. E.g. one property that could enable arrows, mouse wheel, drag, etc. How will this work with a Gamepad?

Reply:
I [changed UpDownArrows to UpDownButtons](https://github.com/microsoft/microsoft-ui-xaml-specs/commit/d0fd39f0c7686bde60b9bd5508d1fb6b65d949ab) to clarify. Thank you for this callout.

Reply:
> I [changed UpDownArrows to UpDownButtons](https://github.com/microsoft/microsoft-ui-xaml-specs/commit/d0fd39f0c7686bde60b9bd5508d1fb6b65d949ab) to clarify. Thank you for this callout.

@SavoySchuler  Fabric Web calls them **SpinButton**, I think Win32 called them **Up-Down Controls** or **Spin Controls**
![image](https://user-images.githubusercontent.com/7389110/58665150-24bff080-8328-11e9-9c06-e99b8f721a51.png)
https://docs.microsoft.com/en-us/windows/desktop/uxguide/ctrl-spin-controls

Reply:
Unresolving, will update this.

Reply:
[Updated to SpinButton.](https://github.com/microsoft/microsoft-ui-xaml-specs/commit/7ecc889b517c0b89839df29b441e2427a47cd8cf)